framework/doc/content/source/preconditioners/VariableCondensationPreconditioner.md

```diff
@@ -0,0 +1,129 @@
+\begin{equation}
+\Delta \lambda = D^{-1}(\Delta r_{2,c}-K_{2,ci} \Delta u_{2,i}-K_{2,cc} \Delta u_{2,c}).
+\label{eqn:condensation_of_LMs}
+\end{equation}
+
+By substituting [eqn:condensation_of_LMs] into [eqn:original_global_matrix], we obtain a simplified linear system of equations that  contains only the primal variable DOFs,
+
+\begin{equation}
+	\begin{bmatrix}
+  K_{1, ii} &  K_{1, ic} & & \\
+  K_{1, ci} &  K_{1, cc} & -MD^{-1}K_{2,ii} &  -MD^{-1}K_{2,cc} \\
+  &  & K_{2, ii} &  K_{2, ic} \\
+  & M^{\intercal}&  &  D^{\intercal}\\
+	\end{bmatrix}
+	\begin{bmatrix}
+\Delta u_{1,i} \\ \Delta u_{1,c} \\ \Delta u_{2,i} \\ \Delta u_{2,c}
+	\end{bmatrix}
+=
+	\begin{bmatrix}
+\Delta r_{1,i} \\ \Delta r_{1,c} - MD^{-1}\Delta r_{2,c} \\ \Delta r_{2,i} \\ 0
+	\end{bmatrix}
+\label{eqn:condensed_global_matrix}
+\end{equation}
+This condensed system (i.e., [eqn:condensed_global_matrix]) is positive definite and therefore state-of-art iterative solution techniques, such as multigrid methods, are applicable. As a post-processing step, the Lagrange multipliers can be recovered from the displacement following [eqn:condensation_of_LMs].
+
+### Dual Basis Function
+
+The computation of $D^{-1}$ in [eqn:condensation_of_LMs] and [eqn:condensed_global_matrix] can be reduced by using +dual basis+ for the Lagrange multipliers. The +dual basis+ is a relative definition to the +standard basis+, which satisfies a biorthogonal condition as follows
+\begin{equation}
+\label{eqn:local_biorthogonal_condition}
+	\int_{\Gamma_e} \psi_j (\boldsymbol{x}) \phi_k(\boldsymbol{x})~\text{d}s = \delta_{jk} \int_{\Gamma_e} \phi_k(\boldsymbol{x})~\text{d}s,
+\end{equation}
+where $\phi_k(\boldsymbol{x})$ is the standard shape function, $\psi_j (\boldsymbol{x})$ is the dual shape function, and $\delta_{jk}$ is the Kronecker delta function. This biorthogonal condition can be assumed to hold in every lower-dimensional element $\Gamma_e$. Owing to the biorthogonality property of the +dual basis+ functions, the integral matrix $D$ become strict diagonal. Thus computation of $D^{-1}$ and condensation steps in [eqn:condensation_of_LMs] and [eqn:condensed_global_matrix] become trivial.
+
+!alert note title=Use of dual basis is optional but recommended
+In order to reduce the computational cost brought by the condensation steps (see [eqn:condensation_of_LMs] and [eqn:condensed_global_matrix]), we recommend the usage of +dual basis+ function for the Lagrange multipliers. Meanwhile, we suggest setting the option `is_diagonal = true` in the preconditioning block to let VCP take the diagonal structure of the couple matrix ($D$) into account for improved efficiency.
+
+One can enable the usage of +dual basis+ by enabling `use_dual = true` in the `Variables` block:
+
+```
+[Variables]
+ [./lm]
+   order = FIRST
+   family = LAGRANGE
+   use_dual = true
+ [../]
+[]
+```
+
+Or setting the `use_dual = true` in the `Contact` block for mortar-based contact:
```

Comment:
For the settings used in this example, dual basis functions are used even if `use_dual = true` is omitted. You could mention that using dual bases with mortar is the default option.

Reply:
Yes, that's right. Let me change that description.